libpod/oci_conmon_linux.go

```diff
@@ -1051,6 +1051,16 @@ func (r *ConmonOCIRuntime) createOCIContainer(ctr *Container, restoreOptions *Co
 		}
 	}
 
+	if l, ok := os.LookupEnv("LISTEN_FDS"); ok {
+		listenFds, err := strconv.ParseUint(l, 10, 64)
+		if err != nil {
+			logrus.Warnf("Error LISTEN_FDS environment variable not an int %v", err)
```

Comment:
Would adding the container id to the warning provide context?

Reply:
Added